mita/util.py

```diff
@@ -216,8 +216,9 @@ def from_offline_executor(node):
     # by splitting on possible use of __IP'
     match = get_key(configured_nodes, node)
     if match is None:
-        node = node.split('__')[0]
-        match = node if node in configured_nodes else None
+        if node is not None:
```

Comment:
just saying: 

    if node:

Would suffice here. I don't think there is any gain in checking for `None`